### ISSUE TITLE
godot fail to start when `preload` is reformatted - newline problem

### ISSUE THREAD

**Issue by VlachJosef** [Scony/godot-gdscript-toolkit#79]
Having `preload` which reach over 100 character like so:
```
onready var magicPlatform_packed: PackedScene = preload("res://scenes/magic-platform/MagicPlatform.tscn")
```

When formatted by `gdformat` ends up look like this:
```
onready var magicPlatform_packed: PackedScene = preload(
	"res://scenes/magic-platform/MagicPlatform.tscn"
)
```

Which subsequently crash godot when launching:

```
arguments
0: /Applications/Godot.app/Contents/MacOS/Godot
1: --path
2: /Users/pepa/godot/platformer/
Current path: /Users/pepa/godot/platformer/scenes/world
Godot Engine v3.2.2.rc2.official - https://godotengine.org
OpenGL ES 3.0 Renderer: Intel Iris OpenGL Engine
 
Registered camera FaceTime HD Camera with id 1 position 0 at index 0
SCRIPT ERROR: GDScript::reload: Parse Error: Error parsing expression, misplaced: '\n'
   At: res://scenes/world/Player.gd:7.
ERROR: reload: Method failed. Returning: ERR_PARSE_ERROR
   At: modules/gdscript/gdscript.cpp:599.

```

**Comment by Scony** [comment]
Apparently `gdformat` exposed another GDScript bug. I'll confirm and make WA till vnen fixes GDScript in 4.0.

**Comment by Scony** [comment]
actual fix provided in #89 by @lyuma
